src/sensor.h

```diff
@@ -95,6 +96,11 @@ namespace librealsense
                 = std::make_shared< recommended_proccesing_blocks_snapshot >( get_recommended_processing_blocks() );
         }
 
+        rsutils::subscription register_options_value_changed_callback( options_watcher::callback && cb ) override
+        {
+            throw not_implemented_exception( "Registering options value changed callback is not implemented for this sensor" );
```

Comment:
Isn't this already provided in sensor_interface?

Reply:
No, the interface is pure virtual